src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -185,8 +185,7 @@ define(function (require, exports, module) {
             callback = options;
         }
         options = options || {};
-
-        var encoding = options.encoding || "utf8";
+        options.encoding = options.encoding === null ? null : "utf8";
```

Comment:
So if encoding is not set, `options.encoding` is `undefined` which is `!== null` so it will default to `utf8`. Is that what we want here?